### DISCUSSION TITLE
Fixing rabbitmqctl-status on v 3.8.2

### DISCUSSION THREAD

**Discussion by drencrom** [rabbitmq/rabbitmq-server#7210] [Questions]
I've trying to apply this patch https://github.com/rabbitmq/rabbitmq-server/pull/4897 to the rabbitmq-server package for Ubuntu Focal (3.8.2) but to make it work I had to change the unknown free space value from `:unknown` to `:undefined`.
I can't find the reason why this is so and maybe it is still broken in the last 3.8 if nothing else has changed.
Any help would be appreciated.

**Comment by michaelklishin** [comment]
I could not find any relevant changes besides https://github.com/rabbitmq/rabbitmq-server/pull/6294. `3.8.2` is from 3+ years ago but I don't think disk monitor return value has changed in a long time.

**Comment by drencrom** [comment]
Thank you for you answer but #6294 can't be the reason the patch does not work because it was never merged into the 3.8 branch.

**Reply by michaelklishin** [reply]
I did not imply that it was the reason, only that I do not remember other relevant changes. `:undefined` sounds OK as long as CLI tools in 3.8 know how to format it.
